provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,70 @@
+
+## Add a Workflow Repository
+
+1. **Click Automation** > **Embedded Workflows** > **Repositories**
+2. **Click Configuration** > **Add new Repository**
+3. Provide the `Name` and `URL` for the repository:
+   - `Name`: Local name to identify this repository. Sample: `Example Workflows`
+   - `Url`: Git repository URL. Sample: `https://github.com/ManageIQ/workflows-examples`
+4. Click **Save**
+
+## Create a Provisioning Dialog
+
+1. Click **Automation** > **Embedded Automate** > **Customization**
+2. Click **Service Dialogs** > **All Dialogs**
+3. Click **Configuration** > **Add a new Dialog**
+4. Provide a `Name` and `Description` for the dialog:
+   - Name: Name used in the Service Catalog. Sample: `sm-med-lg dialog`
+   - Description: More information. Sample: `Dialog choosing vm size`
+5. Add a text box named `vm_name`.
+6. Add a Dropdown named `size`.
+7. Click the edit pencil icon for the "size" dropdown.
+8. Under **Options** > **Entries** enter the drop down values:
+   - Name=Large Value=large
+   - Name=Medium Value=medium
+   - Name=Small Value=small
+
+9. Click **Save**
+
+## Create a Service Catalog Item
+
+1. Click **Service** > **Catalogs**
+2. Click **Catalogs** > **All Catalogs**
+3. Click **Configuration** > **Add a New Catalog**
+4. Provide a `Name` and `Description`:
+   - `Name`: Name displayed in the catalog. Sample: `Provision Vm`
+   - `Description`: More information. Sample: `Provisioning Vm by size`
+5. Click **Save**
+6. Click **Catalog Items** > **All Catalog Items** > **Provisioning**
+7. Click **Configuration** > **Add a New Catalog Item**
+8. Choose `VmWare` as the value for `Catalog Item Type`
+9. Under the **Basic Information** Tab, provide the default VM values:
+   - `Name`: Name displayed in the catalog. Sample: `Provision Vm with Size`
+   - `Description`: More information. Sample: `Provision a vm choosing size`
+   - `Display in Catalog`: You always want this checked.
+   - `Catalog`: Organization concept. Sample: `My Company/Provisioning`
+   - `Dialog`: The dialog created above. Sample: `sm-med-lg dialog`.
+   - `Provisioning Entry Point`: `Embedded Workflow, provision-vm-small-med-large.asl`
+17. Under the **Request Info** > **Catalog** Tab enter the following values:
+   - In the `A Naming` section, Enter any value for `VM Name`. It is overwritten by our dialog value.
+20. Under the **Environment** Tab, choose a `Host` and `Datastore`:
+    - `Host`: Host that runs the VM.
+    - `Datastore`: Disk that stores the VM.
+22. Under the **Networking** Tab, poluate the `Network Adapter`. 
+24. Click **Add**
+
+## Provisioning a Virtual Machine
+
+1. Click **Services > **Catalogs** > **Service Catalogs**
+2. Click **Order** Button under the newly created service, "Vm with Size"
+3. Enter a `VM name` and `size`. Sample: `Vm Name = demo-1` and `Size = small`.
```

Comment:
Same comment as Line 16

Reply:
Provide the following fields:

- *VM Name*: Name of the VM. For example, `demo-1`.
- *size*: Size of the VM. Size can be one of `small`, `medium` or `large`. For example `small`.